yarn.lock

```diff
@@ -22,6 +22,11 @@
   resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.21.4.tgz#457ffe647c480dff59c2be092fc3acf71195c87f"
   integrity sha512-/DYyDpeCfaVinT40FPGdkkb+lYSKvsVuMjDAG7jPOWWiM1ibOaB9CXJAlc4d1QpP/U2q2P9jbrSlClKSErd55g==
 
+"@babel/compat-data@^7.21.5":
```

Comment:
Why is yarn.lock edited?

Reply:
Weird. Presumably the answer is `git commit -am "..."` and I just missed seeing this. Let me yank it.

Reply:
Force pushed with a version that has only the blueprint changes.

Reply:
And #10271 will prevent this happening to other folks.